background/background.go

```diff
@@ -66,8 +64,7 @@ func blockConnected() {
 
```

Comment:
can blockConnected() take shutdownCtx?

Reply:
See comment on `checkWalletConsistency()` - we want the RPC clients to be available after a shutdown signal is received so an in-progress `blockConnected()` func can complete its work before vspd shuts down.